src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExercise.java

```diff
@@ -500,38 +500,7 @@ public void filterSensitiveInformation() {
     @Override
     public Set<Result> findResultsFilteredForStudents(Participation participation) {
         boolean isAssessmentOver = getAssessmentDueDate() == null || getAssessmentDueDate().isBefore(ZonedDateTime.now());
```

Comment:
I don't see where isAssessmentOver is used. Maybe I#m blind because codacy didn't report anything 😄

Reply:
Jupe you are right, this can be removed as it is already called in `checkForRatedAndAssessedResult`. Good catch! :)